lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -360,6 +361,13 @@ func (handler *ContainersHandlersImpl) GetContainerLogsHandler(params containers
 		return containers.NewGetContainerLogsInternalServerError()
 	}
 
+	// containers with PluginVersion > 0 will use updated output logging on the backend
+	// containers with ExecConfig.Version = nil are assumed to be PluginVersion 0
+	if container.ExecConfig.Version != nil && container.ExecConfig.Version.PluginVersion > 0 {
```

Comment:
@caglar10ur would you mind if this was captured in an issue and approached at a later date? Time constraints and all..